src/app.py

```diff
@@ -1,13 +1,22 @@
-import sys
-from PyQt5.QtWidgets import QApplication
-
-from widgets.main_layout import RandomGenerator
+from database.database_writer import Writer
+from database.database_reader import Reader
+from database.database_data_updater import DocumentUpdater
```

Comment:
```suggestion
from database import Writer, Reader, DocumentUpdater
```
ale w sumie to i tak jest tylko do testu, więc okej